src/CommandingOfficers/BuggedEnergyCostCommander.java

```diff
@@ -0,0 +1,60 @@
+package CommandingOfficers;
+
+import Engine.GameScenario;
+import Engine.UnitMods.UnitDamageModifier;
+import Engine.UnitMods.UnitDefenseModifier;
+import Engine.UnitMods.UnitModifier;
+import Terrain.MapMaster;
+import java.util.ArrayList;
+import CommandingOfficers.CommanderInfo.InfoPage;
+
+/** Base class for AW2/3 COs (AW1 is special) */
+public abstract class BuggedEnergyCostCommander extends Commander
```

Comment:
This is just gratuitous. The slightly-inaccurate `Trilogy` was a better prefix I think, especially as you aren't even implementing the AW1 version (yet?)

Reply:
Yes, "yet" is correct.
You did say you want to read _accurate_ things.

Reply:
...sigh. I mean yes, but I also prefer to read meaningful things. This new name gives no indication of why someone would want to use this class (Hmm, I want my new commander to have incorrect power costs. Oh look, a convenient base class!). Trilogy does, though it would also create confusion given that AW1/4 have their own base classes already (and so do 2 and 3 for that matter), but a little skulking around would clarify things. As usual, my vote is going to be for something annoyingly wordy like `AW2And3CommanderBase` or some such tomfoolery, but I'd live with `TrilogyCommander` too.